src/DependencyInjection/StorybookExtension.php

```diff
@@ -139,6 +140,17 @@ static function (ChildDefinition $definition, AsComponentMock $attributeInstance
         $container->register('storybook.component_mock_subscriber', ComponentMockSubscriber::class)
             ->setArgument(0, new Reference('storybook.component_proxy_factory'))
             ->addTag('kernel.event_subscriber');
+
+        // Maker
+        $container->register('storybook.maker.story_renderer', \Storybook\Maker\StoryRenderer::class)
+            ->setArgument(0, new Reference('maker.generator'))
+            ->setArgument(1, new Reference('ux.twig_component.component_factory'))
+            ->setArgument(2, '%kernel.project_dir%/stories');
```

Comment:
Stories path depends on Storybook configuration, so I think this should be handled in a interactive prompt at maker level, with _smart_ suggestions like `stories/` or the current component template directory?